token-minter/sources/modules/collection_properties.move

```diff
@@ -0,0 +1,338 @@
+module minter::collection_properties {
```

Comment:
does this module need generic Type?

Reply:
Yes it does, it can either take in `Object<Collection>` or `Object<CollectionProperties>`. I had to change this as this was needed for migration of the contracts.